spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/PluginLoader.scala

```diff
@@ -0,0 +1,65 @@
+/*
```

Comment:
It's `plugings-api` module and it contains only the contract interfaces between Enceladus and its plugins. And I wanted this module to be as thin as possible. The way the plugins are loaded is not a API's concern.
On the other hand, plugins are designed to be used in Spark Jobs so that's why they are at that module.

Reply:
Yeah, what you write is logical. My mistake. :-)